errors/daemon.go

```diff
@@ -872,4 +863,31 @@ var (
 		Description:    "An attempt to create a volume using a driver but the volume already exists with a different driver",
 		HTTPStatusCode: http.StatusInternalServerError,
 	})
+
+	// ErrorCodeCantStart127 is generated when contained cmd can't start,
+	// contained command not found error, exit code 127
+	ErrorCodeCantStart127 = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "COMMANDNOTFOUND",
```

Comment:
The pattern we're following is that the "Value" matches the ErrorCode, but in all caps. So this would be CANTSTART127